Tasks/DockerInstallerV0/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 171,
+        "Minor": 172,
```

Comment:
These changes should reflect on `task.loc.json` as well.
Consider building the task locally

Reply:
@ds-ms Included missing file (skipped by mistake)